packages/apps/tools/package.json

```diff
@@ -76,16 +76,15 @@
     "@vanilla-extract/jest-transform": "1.1.1",
     "@vanilla-extract/next-plugin": "2.3.1",
     "@walletconnect/types": "~2.8.1",
+    "@webpro/next-translate-plugin": "^2.6.3",
```

Comment:
?

Reply:
- Ugly: https://github.com/kadena-community/kadena.js/pull/1018/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L14
- Fix: https://github.com/aralroca/next-translate-plugin/pull/55

tl/dr; that plugin doesn't support custom base path cq monorepos so I fixed it. We'll go back to the original once it's sorted downstream.